dicoogle/src/main/resources/webapp/js/components/search/result/studyView.js

```diff
@@ -113,12 +113,12 @@ const StudyView = createReactClass({
   handleSelect(item) {
     let { studyInstanceUID } = item;
     // ResultSelectActions.select(item);
-    let value = this.refsClone[studyInstanceUID].getChecked();
+    let value = this.refsClone[studyInstanceUID].checked;
     if (value) ResultSelectActions.select(item, studyInstanceUID);
     else ResultSelectActions.unSelect(item, studyInstanceUID);
   },
-  handleRefs: function(id, input) {
-    this.refsClone[id] = input;
+  handleRefs: function(id, input) {	
+    this.refsClone[id] = input;	
```

Comment:
Odd. Is this trailing whitespace? I thought prettier was in place to fix it automatically.

Reply:
Me too. It's heavy to have those pre commit hooks, so at least they can make some magic for us. It does not seems to be the case, here.